cli/parse_global_options_for_fuzzer.cpp

```diff
@@ -58,6 +73,14 @@ FuzzerArgs ParseGlobalOptionsForFuzzer(GlobalArgs &global_args, GlobalFuzzerOpti
         ("out_dir,o", 
             po::value<std::string>(&global_options.out_dir), 
             "Set output dir. Default is `/tmp/fuzzuf-out_dir`.")
+        ("executor,e",
+            po::value<fuzzuf::cli::ExecutorKind>(&global_options.executor)->default_value(global_options.executor),
+            "Specify fuzzing executor. Default is `native`.")
+        ("exec_path",
+        global_options.exec_path ?
+            po::value<std::string>()->default_value(global_options.exec_path->string()) :
+            po::value<std::string>()->default_value(""),
```

Comment:
（純粋な疑問）三項演算子を使わないと意図通りにならないのですか？
`po::value<std::string>()->default_value(global_options.exec_path->string())` の1行ではダメ？

Reply:
This is because `exec_path` is `std::optional<fs::path>`, which can be `std::nullopt`. The ternary operator is required in such case, or the CLI fails to dereference the value e.g.:
```
# ./fuzzuf
[!] std::bad_alloc
```